src/Analyzers/CSharp/Tests/UseCollectionExpression/UseCollectionExpressionForArrayTests.cs

```diff
@@ -52,7 +52,7 @@ class C
                     int[] i = [1, 2, 3];
                 }
                 """,
-            LanguageVersion = LanguageVersionExtensions.CSharpNext,
+            LanguageVersion = LanguageVersion.CSharp12,
```

Comment:
Why do we pick `CSharp12` here vs. `LatestMajor`?

Reply:
The change from CSharpNext to CSharp12 is designed to be mechanical, so not a lot of scrutiny was given to individual tests (I expect that would have been discussed when the test was written).
This specific test is call `TestInCSharp12`, so CSharp12 seems right.